src/index.html

```diff
@@ -17,11 +20,306 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.scss"
```

Comment:
The stylesheet link is pointing to a SCSS file. Ensure that this is compiled to CSS and update the link to point to the compiled CSS file.